app-backend/api/src/main/scala/exports/Routes.scala

```diff
@@ -101,4 +107,19 @@ trait ExportRoutes extends Authentication
       completeSingleOrNotFound
     }
   }
+
+  def exportFiles(exportId: UUID): Route = authenticate { user =>
+    rejectEmptyResponse {
+      val result: OptionT[Future, List[String]] = for {
```

Comment:
I'm not sure this much logic belongs in the router.  We've tended to keep the route definitions pretty simple, basically just running logic that exists elsewhere, usually on the relevant database tables.